src/main/java/io/sudhakar/student/urldetails/UrlClass.java

```diff
@@ -0,0 +1,30 @@
+package io.sudhakar.student.urldetails;
+
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@Getter
+public class UrlClass {
```

Comment:
EmployeeEndPointDetails

Reply:
changed class name  to EmployeeEndPointDetails